Patch by Robert Scheck <robert@fedoraproject.org> for ssldump >= 0.9b3, which
replaces the inclusion of <net/bpf.h> by <pcap-bpf.h> because of changed files.
It adds some 64 bit support in ./configure for lib64 directories around libpcap
and ensures that dynamic linking to libpcap is possible.

--- ssldump-0.9b3/base/pcap-snoop.c		2010-01-23 00:30:24.000000000 +0100
+++ ssldump-0.9b3/base/pcap-snoop.c.libpcap	2010-01-23 00:34:11.000000000 +0100
@@ -49,7 +49,7 @@
 
 #include <pcap.h>
 #include <unistd.h>
-#include <net/bpf.h>
+#include <pcap-bpf.h>
 #ifndef _WIN32
 #include <sys/param.h>
 #endif
--- ssldump-0.9b3/configure.in			2001-11-26 23:38:13.000000000 +0100
+++ ssldump-0.9b3/configure.in.libpcap		2010-01-23 00:33:12.000000000 +0100
@@ -62,7 +62,7 @@
 dnl Look for PCAP
 dnl We absolutely need pcap
 ac_pcap_inc_dir="/usr/include /usr/include/pcap /usr/local/include"
-ac_pcap_lib_dir="/usr/lib /usr/local/lib"
+ac_pcap_lib_dir="/usr/local/lib64 /usr/local/lib /usr/lib64 /usr/lib"
 
 AC_ARG_WITH(pcap,[--with-pcap		  root location for pcap library],
 	if test "$withval" = "no"; then
@@ -102,13 +102,13 @@
 AC_MSG_CHECKING(for PCAP library)
 ac_found_pcap_lib_dir="no"
 for dir in $ac_pcap_lib_dir; do
-	if test -f $dir/libpcap.a; then
+	if test -f $dir/libpcap.a -o -f $dir/libpcap.so; then
 	dnl Ok, we think we've found them, but check that they
 	dnl actually ontain the right functions
 		save_LIBS=$LIBS
 		save_LDFLAGS=$LDFLAGS
 		LIBS="-lpcap $LIBS"
-		if test "$dir" != "/usr/lib"; then
+		if test "$dir" != "/usr/lib" -a "$dir" != "/usr/lib64"; then
 			LDFLAGS="-L$dir $LDFLAGS"
 		fi
 		AC_TRY_LINK_FUNC(pcap_open_live,ac_linked_libpcap="true",
